CHANGELOG.md

```diff
@@ -2,19 +2,28 @@
 
 ## Breaking Changes
 
+* `encryptionRequired` for `POST/PUT /services` is now a required field
```

Comment:
personally i'm not a big fan of line breaking in markdown. it also made this PR a bit harder to spot check.

Reply:
Hmmm, something must have formatted it. I didn't do it on purpose.